### PR TITLE
feat: added service & ingress to expose Traefik /ping endpoint

### PR DESCRIPTION
**Pull request by bancey**
### Jira link (if applicable)
N/A

### Change description ###
Added service & ingress to expose the traefik /ping endpoint so DARTs can use it via a health probe on their Load Balancer to be used for the DARTs Proxy.

Tested in sandbox.

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### base/kustomization.yaml
- Added a new resource `traefik2/demo/traefik-ping-ingress.yaml` under resources.

### traefik2/demo/00-traefik2.yaml
- Added port configuration for traefik exposing internal service with a specific load balancer IP.
- Added an additional internal service with specific labels for the traefik deployment.

### traefik2/demo/01-traefik2.yaml
- Added port configuration for traefik exposing internal service with a specific load balancer IP.
- Added an additional internal service with specific labels for the traefik deployment.

### traefik2/demo/traefik-ping-ingress.yaml
- Added a new file defining an Ingress resource for traefik's internal ping service.

### CODE REVIEW

**Comment by cakeben** [comment]
I have absolutely no concerns with this platform wide change to benefit DARTS

**Review by cakeben — APPROVED** [review]

File: apps/admin/traefik2/demo/traefik-health-endpoint.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+kind: Service
```
Comment (thomast1906) [inline apps/admin/traefik2/demo/traefik-health-endpoint.yaml:2]: can the service be reviewed? There is already a traefik service in place on admin namespace

```
kubectl get svc                     
NAME      TYPE         
traefik   LoadBalancer
```

If this is needed, you need to add a patch for the svc

File: apps/admin/traefik2/demo/kustomization.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- ../../base
+  - ../../base
+  - ./traefik-health-endpoint.yaml
```
Comment (thomast1906) [inline apps/admin/traefik2/demo/kustomization.yaml:5]: that a typo?

Comment (bancey) [reply apps/admin/traefik2/demo/kustomization.yaml]: Sorry @thomast1906  thought I'd reverted the changes to that file already, sorted that now

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**